src/ui/pages/home.page.ts

```diff
@@ -10,6 +10,23 @@ export class HomePage extends SalesPortalPage {
 
   uniqueElement = this.title;
 
+  ordersThisYearMetric = this.page.locator(
+    "#total-orders-container .card-text.display-6.text-primary"
```

Comment:
Выглядит не очень стабильным селектор
```suggestion
    "#total-orders-container p"
```